klientjs/core
# [OPEN-STACK] Update to 1.7.0

# OpenStack update report ( [1.6.1](https://github.com/klientjs/open-stack/releases/tag/1.6.1) :arrow_right: [1.7.0](https://github.com/klientjs/open-stack/releases/tag/1.7.0) )

> Please check below the complete report which is a summary of changes made.
> It can help you to know what changes must be made manually.

## scripts

#### SKIPPED

  - lint
  - lint:fix
  - prettier
  - prettier:fix
  - test
  - dist
  - dist:cjs
  - dist:esm
  - pre-commit
  - check
  - update:dependencies
  - update:open-stack
  - coverage:badge
  - coverage:open
  - coverage:serve
  - prepare
  - prepare:reset
  - prepare:husky
  - prepare:pre-commit
  - prepare:commit-msg
  - release
  - configure

## devDependencies

#### UPDATED

  - [@release-it/conventional-changelog](https://www.npmjs.com/package/@release-it/conventional-changelog)
  - [release-it](https://www.npmjs.com/package/release-it)

#### SKIPPED

  - [@commitlint/cli](https://www.npmjs.com/package/@commitlint/cli)
  - [@commitlint/config-conventional](https://www.npmjs.com/package/@commitlint/config-conventional)
  - [@klient/open-stack-cli](https://www.npmjs.com/package/@klient/open-stack-cli)
  - [@types/jest](https://www.npmjs.com/package/@types/jest)
  - [@types/node](https://www.npmjs.com/package/@types/node)
  - [@typescript-eslint/eslint-plugin](https://www.npmjs.com/package/@typescript-eslint/eslint-plugin)
  - [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser)
  - [auto-changelog](https://www.npmjs.com/package/auto-changelog)
  - [commitlint](https://www.npmjs.com/package/commitlint)
  - [conventional-changelog-conventionalcommits](https://www.npmjs.com/package/conventional-changelog-conventionalcommits)
  - [eslint](https://www.npmjs.com/package/eslint)
  - [eslint-config-airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base)
  - [eslint-config-airbnb-typescript](https://www.npmjs.com/package/eslint-config-airbnb-typescript)
  - [eslint-config-prettier](https://www.npmjs.com/package/eslint-config-prettier)
  - [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import)
  - [eslint-plugin-unused-imports](https://www.npmjs.com/package/eslint-plugin-unused-imports)
  - [husky](https://www.npmjs.com/package/husky)
  - [jest](https://www.npmjs.com/package/jest)
  - [npm-check-updates](https://www.npmjs.com/package/npm-check-updates)
  - [prettier](https://www.npmjs.com/package/prettier)
  - [ts-jest](https://www.npmjs.com/package/ts-jest)
  - [ts-node](https://www.npmjs.com/package/ts-node)
  - [tsutils](https://www.npmjs.com/package/tsutils)
  - [typescript](https://www.npmjs.com/package/typescript)

## Files

#### UPDATED

  - .release-it.json

#### SKIPPED

  - .eslintrc
  - .prettierrc
  - .editorconfig
  - .gitignore
  - .commitlintrc
  - jest.config.json
  - tsconfig.json
  - CODE_OF_CONDUCT.md
  - CONTRIBUTING.md
  - LICENCE
  - .github/PULL_REQUEST_TEMPLATE.md
  - .github/ISSUE_TEMPLATE/config.yml
  - .github/ISSUE_TEMPLATE/4_Help.yaml
  - .github/ISSUE_TEMPLATE/3_Documentation.yaml
  - .github/ISSUE_TEMPLATE/2_Feature_request.yaml
  - .github/ISSUE_TEMPLATE/1_Bug_report.yaml
  - .github/workflows/unpublish.yml
  - .github/workflows/tests.yml
  - .github/workflows/release.yml
  - .github/workflows/publish.yml
  - .github/workflows/open-stack.yml
  - .github/workflows/dependencies.yml
  - .github/workflows/cache.yml

*This message has been auto generated*
